### PR TITLE
Create a Docker image to bundle both stoa and a full node

### PR DESCRIPTION
**Pull request by TrustHenry**
Dockerfile can produce `Stoa` dockerImage, which is based on `agora` dockerImage.
Used `supervisord` to multiProcessing run the Stoa service and `agora Fullnode`

Relates to #28

### CODE REVIEW

**Comment by TrustHenry** [comment]
@Geod24 Could you Review to see if it's all right

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ARG DUB_OPTIONS="-b cov"
```
Comment (Geod24) [inline Dockerfile]: You don't need this

File: config/service_script.conf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[supervisord]  ## This is the main process for the Supervisor
+nodaemon=true  ## This setting is to specify that we are not running in daemon mode
+
+## agora_service
+[program:agora_service]
+command=agora -c config/agora_config.yaml
```
Comment (Geod24) [inline config/service_script.conf]: Does this work ? What WD is used ?

File: config/agora_config.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+################################################################################
```
Comment (Geod24) [inline config/agora_config.yaml]: This file shouldn't be here.

**Review by Geod24 — CHANGES_REQUESTED** [review]

File: config/service_script.conf (hunk 2 above)
Comment (TrustHenry) [reply config/service_script.conf]: What is WD?

Comment (Geod24) [reply config/service_script.conf]: Working directory

File: config/agora_config.yaml (hunk 3 above)
Comment (TrustHenry) [reply config/agora_config.yaml]: So when It run Stoa Docker, do It have to deliver it from the outside?

Comment (Geod24) [reply config/agora_config.yaml]: Yes, see e.g. https://github.com/bpfkorea/agora/blob/e14004eefac4c68fda6127fcae2236bac52b101f/tests/system/docker-compose.yml#L17

File: config/service_script.conf (hunk 2 above)
Comment (TrustHenry) [reply config/service_script.conf]: Yes, it works. 
It's here. /usr/local/bin/agora

File: config/agora_config.yaml (hunk 3 above)
Comment (TrustHenry) [reply config/agora_config.yaml]: The file was changed to an example file.
Now refer to external config file
`docker run -p 127.0.0.1:3836:3836/tcp -v $(pwd)/config/:/config/config.example.yaml -e "CONFIG=config/config.example.yaml" bpfk/stoa`

File: Dockerfile (hunk 1 above)
Comment (TrustHenry) [reply Dockerfile]: Thanks, I Fixed it now.

**Comment by TrustHenry** [comment]
@Geod24 ping

**Comment by TrustHenry** [comment]
Is there anything lacking in this PR?

**Comment by Geod24** [comment]
The Agora config file shouldn't be there

**Comment by TrustHenry** [comment]
> The Agora config file shouldn't be there

Yes, It don't need it, Don't It need an example file for the test?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# From Agora Runner
+FROM bpfk/agora:latest
+RUN apk add --no-cache curl git python3 py-pip alpine-sdk \
+    bash autoconf libtool automake nodejs npm supervisor
+ADD . /root/stoa/
+WORKDIR /root/stoa/
+RUN npm ci
+EXPOSE 3836
+# Use the supervisord to run agora and stoa multiprocessor.
+ENTRYPOINT ["supervisord","-c","/root/stoa/config/service_script.conf"]
```
Comment (Geod24) [inline Dockerfile]: It's better not to install it in the root directory. Use `/stoa/` for the moment (like we use `/agora/`)

Comment (TrustHenry) [reply Dockerfile]: Yes, I fixed it now.

**Comment by TrustHenry** [comment]
The Agora config file has been deleted.
Also, I modified the README.md to match.

**Comment by TrustHenry** [comment]
Ready for review.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**